### ISSUE TITLE
Clicking [+] on empty "Add new term" field submits bad POST request

### ISSUE THREAD

**Issue by jamiefolsom** [mitodl/lore#580]
On the manage taxonomies panel, if you attempt to submit an empty term field, by clicking the plus button, a bad POST request is sent.

![screenshot 2015-09-01 11 53 47](https://cloud.githubusercontent.com/assets/882377/9609209/81a95c96-50a0-11e5-9260-21fadd129721.png)

That POST should just be prevented, and an error shown to the user, saying "Please enter a term to be added." or something.

**Comment by noisecapella** [comment]
You will also see a bad request if you add a duplicate term (I forget if we handle that case already). It's probably best to check the 400 being returned and translate to a more useful error message in those two cases

**Comment by jamiefolsom** [comment]
@noisecapella I agree about checking for, and giving user feedback on, duplicate terms, although both in that case, and even more so in this one (empty term), wouldn't we rather just catch this, and not send any XHR request?

**Comment by noisecapella** [comment]
I have no preference either way
